python/test/unit/language/test_core.py

```diff
@@ -5132,7 +5132,7 @@ def kernel(X, Y, BLOCK: tl.constexpr):
     y = torch.zeros(shape, dtype=torch.float32, device=device)
 
     kernel[(1, )](x, y, BLOCK=shape[0])
-    torch.allclose(getattr(torch, func_str)(x), y, rtol=1e-3)
+    assert torch.allclose(getattr(torch, func_str)(x), y, rtol=1e-3)
```

Comment:
You can use `torch.testing.assert_close` instead

Reply:
Thanks for reviewing this. 
Yes, I knew. I would align with other references of torch.allclose() in the codebase.
or we use np.testing.assert_allclose() like other testcases in this file?

Reply:
hi, @Jokeren, 
It turns out torch.testing.allclose has been [deprecated](https://github.com/pytorch/pytorch/issues/61844). I tried torch.testing.close() too. It's invalid to pass in only rtol. 

How about we use np.testing.all_close() here? Other testcases in the file use it.

Reply:
I never said `torch.testing.allclose`. You can refer back to my post.

What I was suggesting is `torch.testing.assert_close`

Reply:
I can NOT simply replace `torch.allclose` with `torch.testing.assert_close()` here. 
`assert_close` expects both atol and rtol whereas we only have `rtol=1e-3` in the original code. 

I am not very knowledgeable for pytorch. Not sure what atol is appropriate here. Should I dive into np.testing.assert_close to get the same atol here?

Reply:
The only data type you test is float32. I would expect `torch.testing.assert_close(getattr(torch, func_str)(x), y)` to work automatically for you.

> I am not very knowledgeable for pytorch

I would suggest just stepping into the source code and reading the comments of functions